lighthouse-core/report/v2/report-renderer.js

```diff
@@ -199,8 +279,10 @@ window.ReportRenderer = class ReportRenderer {
    * @return {!Element}
    */
   _renderCategory(category) {
-    const element = this._createElement('div', 'lighthouse-category');
-    element.appendChild(this._renderScore(category.score, category.name, category.description));
+    const element = this._dom.createElement('div', 'lighthouse-category');
+    const score = {value: Math.round(category.score), scoringMode: 'numeric'};
```

Comment:
any reason to have this as an object? Seems fine as separate arguments to `renderCategoryScore`/`renderAuditScore` (and you may not even need `scoringMode` in there, at least not yet)